server.js

```diff
@@ -1,35 +1,112 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+import topMusicData from "./data/top-music.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
+const listEndpoints = require('express-list-endpoints');
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
+
+const { Schema } = mongoose;
+
+const songSchema = new Schema ({
+  id: Number,
+  trackName: String,
+  artistName: String,
+  genre: String,
+  bpm: Number,
+  energy: Number,
+  danceability: Number,
+  loudness: Number,
+  liveness: Number,
+  valence: Number,
+  length: Number,
+  acousticness: Number,
+  speechiness: Number,
+  popularity: Number,
+});
+
+const Song = mongoose.model("Song", songSchema);
+
+if(process.env.RESET_DB){
+  const resetDatabase = async () => {
+    await Song.deleteMany();
+    topMusicData.forEach((singleSong) => {
+      const newSong = new Song(singleSong) 
+      newSong.save()
+    }) 
+  }
+  resetDatabase();
+}
+
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  app.get("/", (req, res) => {
```

Comment:
Very neatly presented when opening up the page, I like how the links were clickable!